.travis.yml

```diff
@@ -23,6 +26,10 @@ jobs:
           name: "Ubuntu 16.04 LTS - Clang"
         - os: osx
           osx_image: xcode11.3
+          env: FLAGS='-DUSE_PYTHON_3=OFF -DBoost_NO_BOOST_CMAKE=ON'
+        - os: osx
+          osx_image: xcode11.3
+          env: FLAGS='-DUSE_PYTHON_3=ON'
```

Comment:
This is okay since it's OSX and all OSX builds are allowed to fail.

Reply:
So what's the bottom line for this PR? Add Xenial compilation with + py3 and we're good to go?